test/e2e/components/vl-map-draw-line-action.js

```diff
@@ -1,13 +1,15 @@
 const VlMapDrawAction = require('./vl-map-draw-action');
 
 class VlMapDrawLineAction extends VlMapDrawAction {
-  async draw(c1 = {x: 10, y: 10}, c2 = {x: 20, y: 20}) {
+  async draw(c1 = {x: 152121, y: 212415}, c2 = {x: 152093, y: 212343}) {
```

Comment:
da's gewoon het midden van vlaanderen dan?

Reply:
Ik zie bij de rest dan weer andere coordinaten staan. Het is een beetje vreemd als afnemer om daar willekeurige cijferkes te zien staan. Misschien beter geen default zetten?

Reply:
maakt dat uit voor een test?

Reply:
miss wel de default dan documenteren, zodat gebruikers weten wat dat nu net inhoud. Want nu is het idd een nietszeggend nummer

Reply:
idem voor drawOnCoordinate

Reply:
Het zijn locaties in Antwerpen en dient puur voor afnemers die iets willen tekenen zonder te specificeren waar. Voor deze wijziging was 10, 20 en 20, 20 ook niet veelzeggend.

Reply:
Ik vind het gewoon nu nog wat vreemder ogen: "152121" ziet er gewoon specifieker/vreemder uit dan "10". Maar eigenlijk is 10 evenzeer vreemd om als default te nemen. 
Het is gewoon vreemd om hier uberhaupt een default te hebben want als ik testen schrijf wil ik toch weten waar ik aan het tekenen ben? Want je tekent toch iets om iets te testen, en dan moet je toch weten waar je zit te tekenen? Dan ga je toch altijd als afnemer zelf coordinaten meegeven, anders wijzigt dat morgen naar iets anders (zoals nu) en werken uw testen ineens niet meer.

Dat gezegd zijnde, we winnen er de oorlog niet mee dus als ik den enigste ben die [zaagt en klaagt](https://files.vier.be/soundboardcallboys/S2/Ie-ie-ie2.mp3) hierover moogt ge het gerust laten :)

Reply:
> Want je tekent toch iets om iets te testen, en dan moet je toch weten waar je zit te tekenen? Dan ga je toch altijd als afnemer zelf coordinaten meegeven.

Inderdaad, dus afnemers kunnen best altijd specifiek tekenen. De default is uitsluitend voorzien voor de e2e testen om niet overal een magisch nummer te voorzien. Afnemers kunnen ook gebruik maken van de default wanneer de locatie niet uitmaakt. Documenteren waar de default tekent lijkt me daarom niet nodig, de default verwijderen ook niet.